### DISCUSSION TITLE
I rewrote an integration for vim9. Can I recommend it to more people?

### DISCUSSION THREAD

**Discussion by linjialiang** [PHP-CS-Fixer/PHP-CS-Fixer#8296] [Q&A] [answered]
Address https://github.com/linjialiang/vim-php-cs-fixer

**Comment by Wirone** [comment]
What do you actually expect from us? Code review? I don't use Vim (more than basic stuff), so I can't tell anything about it. You can recommend it anyway 😅.

**Reply by linjialiang** [reply]
@Wirone Can you add a hyperlink to php-cs-fixer's website?

**Reply by Wirone** [reply]
I believe we can't do this, simply because we rather can't verify its value and quality. But maybe @keradus has a different opinion?

**Reply by keradus** [reply, marked as answer]
I do not aim to list each possible integration. I'm OK to list a very small subset of proven integrations that are already well-recognised by the community.

I see your one is very fresh and starting the journey, good luck with growing it!
